monkey/infection_monkey/telemetry/attack/attack_telem.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from infection_monkey.telemetry.base_telem import BaseTelem
+
+__author__ = "VakarisZ"
+
+LOG = logging.getLogger(__name__)
+
+
+class AttackTelem(BaseTelem):
+
+    def __init__(self, technique, status):
+        """
+        Default ATT&CK telemetry constructor
+        :param technique: Technique ID. E.g. T111
+        :param status: ScanStatus of technique
+        """
+        super(AttackTelem, self).__init__()
+        self.technique = technique
+        self.status = status
+
+    telem_type = 'attack'
```

Comment:
1. Why is this a member? We can map types to their string representation and this is more polymorphic
2. Why is this member missing in VictimHostTelem?

Reply:
I get what you're saying. Currently `telem_type` is used in a specific way on monkey island such that various Telem classes on the infection monkey should be mapped to the same `telem_type`.
E.g. VictimHostTelem and AttackTelem should both have the value 'attack'.

I think this is actually good to keep telem_type the same for efficiency reasons because of that monkey island behavior.
However, I can add the class name when I send the telemetry to the island. I'm not sure if mongoengine has good support for it, but it may allow us to define better telemetry models that way, such that they will be based on inheritance.

What do you think?

Regarding 2, that's because it's a child of AttackTelem, so it already has the telem_type property (and it should be the same)

Reply:
About 1: If there's no use case for adding the class name to the telemetry, let's not bother. 
About 2: I didn't know they had to be the same. How about renaming to `telem_category` instead? `type` is a keyword which doesn't really convey what you're trying to say here

Reply:
1. For now no, maybe later on.
2. agreed